pkg/model/fields/fields.go

```diff
@@ -69,8 +69,7 @@ func IsNumeric(v string) bool {
 		Packets,
 		Proto,
 		Bytes,
-		DSCP,
-		TCPFlags:
+		DSCP:
```

Comment:
We should rely on field types in config since these were introduced for doc. 
WDYT ?

We can create a followup for that

Reply:
I'm seeing some discrepancies with the frontend config, so not sure if that would break something or if that's fine. 
Frontend config number fields not mentioned here:
- IcmpType
- IcmpCode
- FlowDirection (probably because it's an index)
- IfDirections (btw, weird to say it's a number since it's an array?)
- PktDropBytes
- PktDropPackets
- PktDropLatestFlags
- DnsFlags

In the other way around, there is "Port" defined here which isn't a formal field...

So maybe it's doable to use frontend config here but that sounds more a work for a PR of its own